code/src/Components/Homepage.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect} from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import { BASE_URL } from 'utils/urls';
+
+import MovieList from 'Components/MovieList';
+import MovieDetails from 'Components/MovieDetails';
+
+// MAIN API FETCH AND ROUTES
```

Comment:
This type of comment could make sense in bigger, more complex files. Here, all that is being done is routing and fetching which is pretty clear when you look at the code, so there's no need to call it out.